card-component-app/UserCard.jsx

```diff
@@ -0,0 +1,28 @@
+const UserCard = (props) => {
+  const [email, setEmail] = React.useState('loading...')
+  const [firstname, setFirstName] = React.useState('loading...')
```

Comment:
Totally your call, but I'd probably be more inclined to use one piece of state here called `user` that is an object with all of the properties that we need. Some people prefer to keep their state more simple/broken out, so it's really just personal preference, but I find it a little easier to read. Also, I'd probably just use a `null` value as the default state, and then conditionally render a loading message if the user isn't present. This can get a little weird with stuff like the `img` src, which will just look like a broken link.  Something like:

```jsx
const [user, setUser] = useState(null);

// Later in the promise callback

setUser({
  firstName: res.data.first_name,
  lastName: res.data.last_name,
  avatar: res.data.avatar,
  email: res.data.email
})

// Finally, when rendering 

{ !user ? <p>Loading...</p> : // render the user info }
```